src/ppx/grfx/grfx_swapchain.cpp

```diff
@@ -248,6 +252,11 @@ Result Swapchain::CreateRenderPasses()
         rpCreateInfo.depthStencilClearValue     = {1.0f, 0xFF};
         rpCreateInfo.ownership                  = grfx::OWNERSHIP_RESTRICTED;
         rpCreateInfo.pShadingRatePattern        = mCreateInfo.pShadingRatePattern;
+        rpCreateInfo.arrayLayerCount            = mCreateInfo.arrayLayerCount;
+#if defined(PPX_BUILD_XR)
+        rpCreateInfo.multiViewState.viewMask        = (mCreateInfo.pXrComponent && mCreateInfo.arrayLayerCount > 1 ? mCreateInfo.pXrComponent->GetDefaultViewMask() : 0);
```

Comment:
super nit/personal preference (same below):

```suggestion
        if (mCreateInfo.pXrComponent && mCreateInfo.arrayLayerCount > 1)
            rpCreateInfo.multiViewState.viewMask = mCreateInfo.pXrComponent->GetDefaultViewMask();
```